docs/send-message.md

```diff
@@ -0,0 +1,131 @@
+both, message transport and message dispatch. There is one command for submitting a message to each of the two
+available bridges, namely `millau-to-rialto` and `rialto-to-millau`.
+
+Submitting a message requires a number of arguments to be provided. Those arguments are essentially the same
+for both submit message commands, hence only the output for `millau-to-rialto` is shown below.
+
+```
+Submit message to given Millau -> Rialto lane
+
+USAGE:
+    substrate-relay send-message millau-to-rialto [OPTIONS] --lane <lane> --millau-host <millau-host> --millau-port <millau-port> --millau-signer <millau-signer> --origin <origin> --rialto-signer <rialto-signer> <SUBCOMMAND>
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+        --fee <fee>
+            Delivery and dispatch fee. If not passed, determined automatically
+
+        --lane <lane>                                        Hex-encoded lane id
+        --millau-host <millau-host>                          Connect to Millau node at given host
+        --millau-port <millau-port>                          Connect to Millau node websocket server at given port
+        --millau-signer <millau-signer>
+            The SURI of secret key to use when transactions are submitted to the Millau node
+
+        --millau-signer-password <millau-signer-password>
+            The password for the SURI of secret key to use when transactions are submitted to the Millau node
+
+        --origin <origin>
+            The origin to use when dispatching the message on the target chain [possible values: Target, Source]
+
+        --rialto-signer <rialto-signer>
+            The SURI of secret key to use when transactions are submitted to the Rialto node
+
+        --rialto-signer-password <rialto-signer-password>
+            The password for the SURI of secret key to use when transactions are submitted to the Rialto node
+
+
+SUBCOMMANDS:
+    help        Prints this message or the help of the given subcommand(s)
+    remark      Make an on-chain remark (comment)
+    transfer    Transfer the specified `amount` of native tokens to a particular `recipient`
+
+```
+As can be seen from the output, there are two types of messages available: `remark` and `transfer`.
+A remark is some opaque message which will be placed on-chain. For basic testing, a remark is
+the easiest to go with.
+
+Usage of the arguments is best explained with an example. Below you can see, how a remark 
+would look like:
+
+```
+substrate-relay send-message millau-to-rialto \
+		--millau-host=127.0.0.1 \
+		--millau-port=10946 \
+		--millau-signer=//Dave \
+		--rialto-signer=//Dave \
+		--lane=00000000 \
+		--origin Target \
+		remark
+```
+Messages are basically regular transactions. That means, they have to be signed. In order
+to send a message, you have to control an account private key on both, the source and
```

Comment:
That is only partially correct :) You only need private key on target chain when `--origin Target` specified (you need to sign that digest you've recently changed). With other origins, private key on target chain isn't required.

I guess there's an issue with relay cli now, though - it always requires target chain key. This probably comes from the moment when `--origin` param has been added. Anyway - we should fix it, until then we may leave this description as-is :)